client/__tests__/actionTests/addUserAction.test.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { addUserRequest } from '../../actions/addUserAction';
+import { ADD_USER_SUCCESS } from '../../actions/actionTypes';
+
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+
+describe('Add user action', () => {
+  it('SHOULD CREATE ADD_USER_SUCCESS ACTION', (done) => {
```

Comment:
### Nitpick
So does _addUserRequest_ create the action "ADD_USER_SUCCESS" or rather dispatches it?